internal/command/launch/describe_plan.go

```diff
@@ -19,7 +19,7 @@ const descriptionNone = "<none>"
 func describePostgresPlan(ctx context.Context, p plan.PostgresPlan, org *api.Organization) (string, error) {
 	provider := p.Provider()
 	switch provider.(type) {
-	case plan.FlyPostgresPlan:
+	case *plan.FlyPostgresPlan:
 		return describeFlyPostgresPlan(ctx, provider.(*plan.FlyPostgresPlan), org)
```

Comment:
the linter wants this to turn into something like
```go
switch provider := p.Provider().(type) {
  case *plan.FlyPostgresPlan:
    return describeFlyPostgresPlan(ctx, provider, org)
```
(assigning the switched condition to a variable of X type, so that we don't have to typecast twice)